docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md

```diff
@@ -24,24 +24,18 @@ This document details the installation steps in order to get up and running quic
 
   * OpenShift https://docs.openshift.org/3.7/install_config/install/planning.html
 
-## Support Matrix for Kubernetes and Openshift 3.7 <a name="support"></a>
+## SPOCK for HPE 3PAR Volume Plugin for Docker
 
-| Platforms                                               | Support for Containerized Plugin | Docker Engine Version | HPE 3PAR OS version              |
-|---------------------------------------------------------|----------------------------------|-----------------------|----------------------------------|
-| Kubernetes 1.6.13                                       | Yes                              | 1.12.6                | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2 |
-| Kubernetes 1.7.6                                        | Yes                              | 1.12.6                | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2 |
-| Kubernetes 1.8.9                                        | Yes                              | 17.06                 | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2 |
-| Kubernetes 1.10.3                                       | Yes                              | 17.03                 | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2 |
-| OpenShift 3.7 RPM based installation (Kubernetes 1.7.6) | Yes                              | 1.12.6                | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2   |
+* [Support Matrix for Kubernetes and Openshift](https://spock.corp.int.hpe.com/spock/utility/document.aspx?docurl=Shared%20Documents/hw/3par/3par_volume_plugin_for_docker.pdf)
 
 **NOTE**
-  * Managed Plugin is not supported for Kubernetes or Openshift 3.7
+  * Managed Plugin is not supported for Kubernetes or Openshift 
 
-  * The install of OpenShift for this paper was done on RHEL 7.4. Other versions of Linux may not be supported.
+  * The install of OpenShift for this paper was done on RHEL 7.x. Other versions of Linux may not be supported.
```

Comment:
Shall we mention 7.x or all supported and qualified versions?

Reply:
its better to call out 7.x., we do support most of the 7.* series of RHEL.